player/public/coreplugins/org.ekstep.nextnavigation-1.0/renderer/plugin.js

```diff
@@ -6,10 +6,14 @@
 Plugin.extend({
 	_type: "nextNavigation",
 	initialize: function () {
+		console.info('Next navigation intialize');
+		this.templateName_top = org.ekstep.pluginframework.pluginManager.resolvePluginResource(this._manifest.id, this._manifest.ver, "renderer/templates/top.html");
```

Comment:
Remove this code & add int he event handler itself. Use single variable "templateName"